admin/assets/javascripts/discourse/models/repo.js

```diff
@@ -94,15 +98,59 @@ export default class Repo {
       }
     }
 
+    if (attributes.plugin) {
+      this.plugin = AdminPlugin.create(attributes.plugin);
+    }
+
     for (const [key, value] of Object.entries(attributes)) {
-      if (key === "latest") {
+      if (["latest", "plugin"].includes(key)) {
         continue;
       }
 
       this[key] = value;
     }
   }
 
+  @cached
+  get nameTitleized() {
+    if (this.plugin) {
+      return this.plugin.nameTitleized;
+    }
+
+    let name = this.name
```

Comment:
I'm just wondering why you need this other stuff? Everything on this page is either Discourse or a plugin isn't it?

Reply:
I needed this since the docker_manager plugin is a hidden plugin and I was fetching plugin information only for the visible plugins. It's no longer the case after the commit https://github.com/discourse/docker_manager/pull/214/commits/662fee0d3f7c390d7fad24aa665db539ac7e000a